lib/rules/func-call-spacing.js

```diff
@@ -86,9 +86,9 @@ module.exports = {
                 return;
             }
 
-            const hasWhitespace = sourceCode.isSpaceBetweenTokens(prevToken, parenToken);
-            const hasNewline = hasWhitespace &&
-                /\n/.test(text.slice(prevToken.range[1], parenToken.range[0]).replace(/\/\*.*?\*\//g, ""));
+            const textBetweenTokens = text.slice(prevToken.range[1], parenToken.range[0]).replace(/\/\*.*?\*\//g, "");
+            const hasWhitespace = /\s/.test(textBetweenTokens);
+            const hasNewline = hasWhitespace && /[\n\r\u2028\u2029]/.test(textBetweenTokens);
```

Comment:
Thanks! Unfortunately, the regexp in `linebreak-style` cannot be centralized in a way that leaves it safe to reuse in recursion: it has a `g` flag, which means, it has a `lastIndex` property that changes every time the `exec` method is called on it to keep track of the last matching position.

We could centralize the pattern as a string and then create the regexp with the `RegExp` constructor. I'd note anyway that those regexes are for slightly different purposes. The one in `func-call-spacing` is meant to match a *LineTerminator*, while the regexp in `line-break-style` finds *LineTerminatorSequences*. Both are defined in http://www.ecma-international.org/ecma-262/6.0/#sec-line-terminators.